### DISCUSSION TITLE
Force loss error

### DISCUSSION THREAD

**Discussion by willphy123** [deepmodeling/deepmd-kit#2278] [Q&A] [answered]
Hi all, 
   Does anyone use siesta DFT data to do training? I tested the H2O example. Below is the loss curve diagram. The loss curve is weird. The force error first decreases and then increases to a large value. Any suggestions will be appreciated. Btw, the loss curve is fine when I use VASP DFT data. 
Thanks 
Will 
<img width="1002" alt="Screen Shot 2023-01-25 at 10 34 54 AM" src="https://user-images.githubusercontent.com/104464250/214652450-0b3056ad-2152-4c44-806c-67a74551dbbc.png">
<img width="1005" alt="Screen Shot 2023-01-25 at 10 34 34 AM" src="https://user-images.githubusercontent.com/104464250/214652453-d75da70d-1b22-4b97-b467-163383120ed5.png">

**Comment by njzjz** [comment, marked as answer]
In this situation, please read [the FAQ](https://docs.deepmodeling.com/projects/deepmd/en/v2.2.0.b0/troubleshooting/precision.html#data) and check your data. Since you have two different levels of data, it is much easier to compare them.

**Reply by willphy123** [reply]
Thanks for your suggestions. Now I solved the problem!
